marshmallow/fields.py

```diff
@@ -994,6 +996,31 @@ def _deserialize(self, value, attr, data):
             self.fail('invalid')
 
 
+class Dict(Field):
+    """A dict field. Supports dicts and dict-like objects.
+
+    .. versionadded:: 2.1.0
+    """
+
+    default_error_messages = {
+        'invalid': 'Not a valid dict.'
```

Comment:
We should try to avoid Python-specific language when possible. marshmallow is most commonly used for HTTP APIs, where clients may not "speak" Python. Maybe "Not a valid mapping type."